reconcile/change_owners/change_types.py

```diff
@@ -389,6 +389,51 @@ def build_change_type_processor(change_type: ChangeTypeV1) -> ChangeTypeProcesso
     return ctp
 
 
+def init_change_type_processors(
+    change_types: Sequence[ChangeTypeV1],
+) -> dict[str, ChangeTypeProcessor]:
+    processors = {}
+    change_type_graph = networkx.DiGraph()
+    for change_type in change_types:
+        change_type_graph.add_node(change_type.name)
+        for i in change_type.inherit or []:
+            change_type_graph.add_edge(change_type.name, i.name)
+        processors[change_type.name] = build_change_type_processor(change_type)
+
+    # detect cycles
+    cycles = list(networkx.simple_cycles(change_type_graph))
+    if cycles:
```

Comment:
nit but can be written as

```python
if cycles := list(networkx.simple_cycles(change_type_graph)):
```

Reply:
it was such a long week that i'm not even ashamed to admit that i was not aware of this syntax :D

Reply:
changed in c28e6fbf5d5836469838817258ef238f4e0d4479